api/common/cors.py

```diff
@@ -43,7 +43,10 @@ def add_cors_headers():
         origin = "https://dynabench.org"
 
     bottle.default_app()
-    bottle.response.headers["Access-Control-Allow-Origin"] = origin
+
+    if not ("Access-Control-Allow-Origin" in bottle.response.headers):
```

Comment:
If the access control allow origin is not set previously (which happens when we hit any non turk endpoint or if we hit a turk endpoint but not in turk mode), then set the header as we were before.

If it is set (which happens when we hit a turk endpoint in turk mode), then set the header as `*`